src/functions/pollUsers/framework/repo/mysql/examiner-repository.ts

```diff
@@ -2,8 +2,9 @@ import { query } from '../../../../../common/framework/mysql/database';
 import { config } from '../../config';
 import * as mysql from 'mysql2';
 import * as moment from 'moment';
+import { StaffDetail } from '../../../../../common/application/models/staff-details';
 
-export const getActiveExaminers = async (): Promise<string[]> => {
+export const getActiveExaminers = async (): Promise<any[]> => {
```

Comment:
What was the reason for changing from `string[]` to `any[]`?

Reply:
It's no longer purely an array of strings. It contains `isLDTM` which is a `boolean`.

Reply:
I should probably enforce it as `StaffDetail[]`.